about/our-team.php

```diff
@@ -266,15 +266,15 @@
                 <!-- Aerospace Division Lead -->
                 <div class="cell-lg-12 bio-area">
                     <div class="range range-sm-middle">
-                        <div class="cell-md-3"><img src="/images/bio-pics/jesse-sullivan.jpg" alt="" width="360" height="404" class="img-responsive" />
+                        <div class="cell-md-3"><img src="/images/bio-pics/joe-moore.jpg" alt="" width="360" height="404" class="img-responsive" />
                         </div>
                         <div class="cell-md-6">
                             <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
-                                <h6 class="h6-with-small"><a href="#"> Jesse Sullivan</a><span class="small text-silver-chalice">Aerospace Division Lead</span></h6>
-                                <p>Jesse is a senior mechanical engineering student interested in all things that fly and go fast. He has 4 years of experience in amateur high-power rocketry and currently holds an L1 certification with the National Association of Rocketry. He founded the Aerospace Division with UNT Robotics in order to foster interest in the hobby and provide a learning experience for anyone to become a part of. </p>
+                                <h6 class="h6-with-small"><a href="#"> Joe Moore</a><span class="small text-silver-chalice">Aerospace Division Lead</span></h6>
+                                <p>Joseph is a Junior Mechanical Engineering student. Formerly a US Marine Master Explosive Ordnance Disposal Technician (EOD), he has always tried to tinker with things and figure out how they work and is passionate about establishing and growing our High-Power Rocketry team into a nationally competitive force. Both through STEM outreach efforts and internal events, he hopes to generate interest and encourage everyone to get into hobby rocketry, and the sport of high power rocketry. </p>
                                 <ul class="list-inline-lg">
                                     <li>
-                                        <a href="mailto:aerospace@untrobotics.com " class="icon icon-sm text-primary fa-envelope"></a>
+                                        <a href="mailto:aerospace-division-lead@untrobotics.com " class="icon icon-sm text-primary fa-envelope"></a>

```

Comment:
Move Jesse to our-alumni page

Reply:
https://www.untrobotics.com/about/our-alumni

Reply:
done